docs/sample_config.yaml

```diff
@@ -33,10 +33,22 @@
 
 ## Server ##
 
-# The domain name of the server, with optional explicit port.
-# This is used by remote servers to connect to this server,
-# e.g. matrix.org, localhost:8080, etc.
-# This is also the last part of your UserID.
+# The public-facing domain of the server
+#
+# The server_name name will appear at the end of usernames and room addresses
+# created on this server. For example if the server_name was example.com,
+# usernames on this server would be in the format @user:example.com
+#
+# In most cases you should avoid using a matrix specific subdomain such as
+# matrix.example.com or synapse.example.com as the server_name for the same
+# reasons you wouldn't use user@email.example.com as your email address.
+# See docs/delegate.md for information on how to host Synapse on a subdomain
```

Comment:
We often link to the exact file on github, just to make it easier for people (especially those that don't install via source).

Reply:
Ok I made that change. i saw a mixture of both so I wasn't sure what was preferred.